hypothesis/viahtml
# Redirect to Checkmate error pages

For: https://github.com/hypothesis/checkmate/issues/15

This removes ViaHTML's own error pages and instead relies on whatever Checkmate tells us to show to people. We do a temporary redirect there for the moment, as we don't have any indication of how long the block should last for.

This cannot be merged until checkmatelib has been updated with: https://github.com/hypothesis/checkmatelib/pull/5. The version number in the requirements is therefore a guess.

### Testing notes

 * Check out checkmatelib as a sibling to this project
 * In checkmatelib run: `git checkout add-html-view`
 * In checkmatelib run `make services dev`
 * Hack the `requirements.txt` to replace the checkmatelib line with `-e ../checkmatelib`
 * Run `make dev`
 * Visit a blocked page: http://localhost:9085/proxy/http://bad.web.app
 * If you have the real rules loaded this should redirect (if not try `bad.example.com` instead)
